packages/amplication-client/src/Workspaces/queries/workspaceQueries.ts

```diff
@@ -66,6 +66,12 @@ export const GET_WORKSPACES = gql`
   }
 `;
 
+export const GET_CONTACT_US_CALENDLY_LINK = gql`
+  query GetContactUsCalendlyLink($id: String!) {
+    contactUsCalendlyLink(where: { id: $id })
```

Comment:
Let's try to avoid methods that are too specific. Maybe we could just call this query `contactUs` and the fields returned can be extended when we need it